code/game/objects/items/devices/paicard.dm

```diff
@@ -140,6 +140,11 @@
 	playsound(loc, 'sound/effects/pai_boot.ogg', 50, 1, -1)
 	audible_message("\The [src] plays a cheerful startup noise!")
 
+	var/turf/T = get_turf(src)
+	for(var/mob/M in get_turf(src))
```

Comment:
this seems absolutely hacky as a solution

Reply:
well how else am i supposed to get the mob holding the card

Reply:
Get the loc of the card, check if it's a mob

Reply:
whats the code for that

Reply:
Figure it out, it's not complicated at all